Tests/SwiftParserTest/translated/ForwardSlashRegexSkippingInvalidTests.swift

```diff
@@ -365,11 +365,12 @@ final class ForwardSlashRegexSkippingInvalidTests: ParserTestCase {
           0
           /x}}1️⃣} /
           2
-        } 
+        2️⃣} 
       }
       """,
       diagnostics: [
-        DiagnosticSpec(message: "extraneous code at top level")
```

Comment:
What was this parsing as previously? Neither are great but the unexpected } seems a little worse